tools/prow-config-updater/config/config.go

```diff
@@ -110,7 +110,14 @@ func UpdateProw(env ProwEnv, dryrun bool) error {
 	return helpers.Run(
 		fmt.Sprintf("Updating Prow configs with command %q", updateCommand),
 		func() error {
-			out, err := cmd.RunCommand(updateCommand)
+			kf := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
+			if kf != "" {
+				authCommand := "gcloud auth activate-service-account --key-file=" + kf
+				if _, err := cmd.RunCommand(authCommand); err != nil {
+					return fmt.Errorf("error activating service account with %q", kf)
+				}
+			}
+			out, err := cmd.RunCommand(updateCommand, cmd.WithEnvs(os.Environ()))
```

Comment:
do we have to pass `cmd.WithEnvs(os.Environ())` every time now? if so I think `cmd` package needs to be fixed

Reply:
We don't have to pass it every time, e.g. the `cp` command does not use it (we don't have to add it here either, but just adding it for safety)
And it does not necessarily have to be `os. Environ()`, it can be any env vars that users want to set for running this command.